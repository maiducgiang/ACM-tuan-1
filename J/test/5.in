3
6,72(40)
52,5(5)
74,32(99)
