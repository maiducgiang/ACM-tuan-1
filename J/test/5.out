16642 2475
473 9
7433 100
